docs/modules/player-lifecycle/custom-player-creation.md

```diff
@@ -0,0 +1,132 @@
+<%(TOC)%>
+# Customising player creation
+
+<%(Callout message="
+Before reading this document, make sure you are familiar with:
+
+  * [Basic player spawning]({{urlRoot}}/modules/player-lifecycle/basic-player-creation)
+  * [Creating entity templates]({{urlRoot}}/reference/concepts/entity-templates)
+  * [Workers in the GDK]({{urlRoot}}/reference/concepts/worker)
+")%>
+
+## Manually sending player creation requests
+
+By default, the Player Lifecycle Feature Module sends a player creation request is sent as soon as the client-worker instance connects to SpatialOS.
+
+To modify this behaviour, you must set `PlayerLifecycleConfig.AutoRequestPlayerCreation` to false by one of the following ways:
+
+* Add an extra `false` argument when adding client systems: `PlayerLifecycleHelper.AddClientSystems(Worker.World, autoRequestPlayerCreation: false);`
+* Set the field to false manually during set-up: `PlayerLifecycleConfig.AutoRequestPlayerCreation = false;`
+
+Once `AutoRequestPlayerCreation` is set to false you must manually call the `RequestPlayerCreation` method, which can be found in the `SendCreatePlayerRequestSystem`.
```

Comment:
Once `AutoRequestPlayerCreation` is set to false, you can call the `SendCreatePlayerRequestSystem.RequestPlayerCreation()` method when you are ready to request player spawning.